backend/index.js

```diff
@@ -14,4 +14,26 @@ app.listen(port,()=>{
     console.log('API is up and running')
 })
 
+app.get('/news/:name', (req,res)=> {
+
+    let currentDate = new Date()
+    let weekAgo = new Date()
+    weekAgo.setDate(currentDate.getDate() - 7)
+    
+    let name = req.params.name
+    let weekAgoString = weekAgo.toISOString().slice(0, 10)
+    let currentString = currentDate.toISOString().slice(0, 10)
+
+    const headers = {headers:{'X-Finnhub-Token':process.env.API_KEY}}
```

Comment:
Can we change the name of the key here. When adding multiple api keys, we want to use very descriptive names.